twopane-navigation/src/components/paneRenderer/PaneRenderer.tsx

```diff
@@ -73,12 +73,23 @@ const PaneRenderer = (props: IPaneRendererProps) => {
         
     }
 
-// TODO MOVE TO ONE RENDER
+    const isAccessible = (component: IPaneComponent) => {
+        const isLastPaneOne = component.pane === paneType.ONE && paneComponents.filter(x => x.pane === paneType.ONE).pop()?.key === component.key;
```

Comment:
nit: what does it mean a pane is 'last'

Reply:
Means it is at the top of the stack, changed.